test/integration/all_data_files_import_test.rb

```diff
@@ -90,50 +93,32 @@ def before_all
 
   test "Category ↔ Attribute relationships are consistent with categories/*.yml" do
     @raw_verticals_data.flatten.each do |raw_category|
-      real_category = Category.find(raw_category.fetch("id"))
-      raw_category.fetch("attributes").each do |friendly_id|
-        property = Property.find_by(friendly_id:)
-        assert_includes real_category.properties, property
-      end
+      properties_via_raw_category_id = Category.find(raw_category.fetch("id")).properties
+      properties_via_raw_attributes = raw_category.fetch("attributes").map { Property.find_by(friendly_id: _1) }
+
+      assert_equal properties_via_raw_attributes.sort, properties_via_raw_category_id.sort
     end
   end
 
   test "Attribute ↔ Value relationships are consistent with attributes.yml when values are listed" do
-    @raw_attributes_data.each do |raw_attribute|
-      property = Property.find(raw_attribute.fetch("id"))
+    @raw_attributes_data.select { _1.key?("values") }.each do |raw_attribute|
+      values_via_raw_id = Property.find(raw_attribute.fetch("id")).property_values
+      values_via_raw_values = raw_attribute.fetch("values").map { PropertyValue.find_by(friendly_id: _1) }
 
-      next unless raw_attribute.fetch("values_from", nil).nil?
-
-      raw_value_friendly_ids = raw_attribute.fetch("values", [])
-      refute raw_value_friendly_ids.empty?
```

Comment:
Can we add another test then to make sure this case is captured? We don't want any attributes that have an empty values array or it tells us something went wrong

Reply:
Ah thought this was just trying to error correct some parsing. An explicitly named test to that effect feels good

Reply:
Added a test for it: https://github.com/Shopify/product-taxonomy/pull/129